exercises/basic_reduce/problem.md

```diff
@@ -40,3 +40,21 @@ function countWords(inputWords) {
 
 module.exports = countWords
 ```
+
+Helps & Hints: 
+The main idea of a reduce function is that you are reducing the values in an array down to a single value.
+  
+This single value is often called the accumulator because it is inside of this variable that values are accumulated.
+
+When you use the javascript array method for reduce, many of the examples have a one return for this method, but it can be easier
+to return the accumulator after performing your work. 
+
+```js
+var sum = [0, 1, 2, 3].reduce(function(accumulator, current_value) {
+  
+    accumulator = accumulator + current_value 
+  
+    return accumulator;
+}, 0);
```

Comment:
For consistency with other code samples, please drop unnecessary semicolons, unnecessary line breaks and use camelCase for variables. :D